swingset-demo/pom.xml

```diff
@@ -22,11 +22,11 @@
 		<version.java>1.8</version.java>
 
 		<version.log4j>2.14.1</version.log4j>
-		<version.swingset>4.0.4</version.swingset>
+		<version.swingset>4.0.5-SNAPSHOT</version.swingset>
```

Comment:
After doing (from parent directory) cd swingset; mvn clean; mvn install. I got an error in swingset-demo. I couldn't find 
`com.nqadmin.swingset:swingset:jar:4.0.5-SNAPSHOT`

Got rid of this "-SNAPSHOT" and things went OK.

Reply:
Removed -SNAPSHOT as suggested.